infra/main.bicep

```diff
@@ -18,9 +18,10 @@ param relativePath string
 param logAnalyticsName string = ''
 param applicationInsightsName string = ''
 param applicationInsightsDashboardName string = ''
+param utcValue string = utcNow()
 
 var abbrs = loadJsonContent('./abbreviations.json')
-var resourceToken = toLower(uniqueString(subscription().id, environmentName, location))
+var resourceToken = toLower(uniqueString(subscription().id, environmentName, location, utcValue))
```

Comment:
I don't think we should add this.

If we run `azd provision` multiple times, it will create multiple KV instances, right?

It's not an expected result.

Reply:
make sense, now that `azd down` will permernently delete kv, so we should remove the time parameter